lib/desktop/settings/pages/genmanagement.dart

```diff
@@ -318,68 +319,69 @@ void _setLanguage(String _selected, BuildContext context) {
 void _setKeyboard(String _selected, BuildContext context) {
   switch (_selected) {
```

Comment:
this is a pretty terrible approach, use a translated string for the keyboard layout name and for keyboard layout use locale toString

Reply:
do you suggest adds a field for layouts, or just use Locales.supported with 'changelanguage' field instead?

I did as the last one in the lastest commit, but for genManagment setLanguage.

Reply:
field for layouts as in? btw there is this lib [flutter_localized_locales](https://pub.dev/packages/flutter_localized_locales) so you can get localized locale name based on the user locale, might be worth a shot

Reply:
I mean, for layouts in the xml, for example adds strings for set others keyboard layouts: <string name="kbrd_layout">English - United States</string>

Reply:
should keyboards layout setup be in this PR?

Reply:
Nope, the keyborad switcher is basically still just a dummy at this point which uses the same list as the language switcher. I was planing to disable the switcher anyways to avoid any false promises